src/pages/CheckoutOverview.page.js

```diff
@@ -0,0 +1,66 @@
+import { priceNumber } from '../HelperFunctions';
+
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class CheckoutOverviewPage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    totalSelector = '.summary_total_label';
+
+    cartItemNameSelector = '.inventory_item_name';
+
+    cartItemDescrSelector = '.inventory_item_desc';
+
+    cartItemPriceSelector = '.inventory_item_price';
+
+    cartTaxSelector = '.summary_tax_label';
+
+    get totalPrice() { return this.page.locator(this.totalSelector); }
+
+    get cartTax() { return this.page.locator(this.cartTaxSelector); }
+
+    get itemName() { return this.page.locator(this.cartItemNameSelector); }
+
+    get itemDescr() { return this.page.locator(this.cartItemDescrSelector); }
+
+    get itemPrice() { return this.page.locator(this.cartItemPriceSelector); }
+
+    async totalPriceCalculated(itemCount) {
+        let total = 0;
+        for (let i = 0; i < itemCount; i++) {
+            const priceString = await this.itemPrice.nth(i).textContent();
```

Comment:
Just FYI you can use:
`const priceStrings = await this.itemPrice.allTextContents();` [allTextContents](https://playwright.dev/docs/api/class-locator#locator-all-text-contents) and then use reduce to calculate total

Reply:
you can try to use it just for the practise or leave it as is. It's totally up to you